docker-versions-create.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#set -x
+# Use this variable to indicate a list of branches that docker hub is watching
+branches_list=('openj9' 'another_name')
```

Comment:
Looking back at your initial findings, it sounded like `openj9-nightly` would be another branch, so I have created and pushed that one too.